fendermint/vm/message/Cargo.toml

```diff
@@ -30,6 +30,7 @@ rand = { workspace = true, optional = true }
 fendermint_vm_encoding = { path = "../encoding" }
 fendermint_vm_actor_interface = { path = "../actor_interface" }
 fendermint_testing = { path = "../../testing", optional = true }
+fendermint_ipc = { path = "../../ipc" }
```

Comment:
Please don't add a reference to something as "heavy" as the IPC crate. The `fendermint_vm_message` create is a dependency for `fendermint_rpc`, so it should be a thin DTO layer concerned only with messages and their basic validation (like signatures), and not drag in the entire tokio stack.